x-pack/test/api_integration/apis/security_solution/events.ts

```diff
@@ -0,0 +1,828 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+
+import {
+  Direction,
+  TimelineEventsQueries,
+} from '../../../../plugins/security_solution/common/search_strategy';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+const TO = '3000-01-01T00:00:00.000Z';
+const FROM = '2000-01-01T00:00:00.000Z';
+
+// typical values that have to change after an update from "scripts/es_archiver"
+const DATA_COUNT = 7;
+const HOST_NAME = 'suricata-sensor-amsterdam';
+const TOTAL_COUNT = 96;
+const EDGE_LENGTH = 25;
+const ACTIVE_PAGE = 0;
+const PAGE_SIZE = 25;
+const LIMITED_PAGE_SIZE = 2;
+
+const FILTER_VALUE = {
+  bool: {
+    filter: [
+      {
+        bool: {
+          should: [{ match_phrase: { 'host.name': HOST_NAME } }],
+          minimum_should_match: 1,
+        },
+      },
+      {
+        bool: {
+          filter: [
+            {
+              bool: {
+                should: [{ range: { '@timestamp': { gte: FROM } } }],
+                minimum_should_match: 1,
+              },
+            },
+            {
+              bool: {
+                should: [{ range: { '@timestamp': { lte: TO } } }],
+                minimum_should_match: 1,
+              },
+            },
+          ],
+        },
+      },
+    ],
+  },
+};
+
+export default function ({ getService }: FtrProviderContext) {
+  const retry = getService('retry');
+  const esArchiver = getService('esArchiver');
+  const supertest = getService('supertest');
+
+  describe('Timeline', () => {
+    before(() => esArchiver.load('auditbeat/hosts'));
+    after(() => esArchiver.unload('auditbeat/hosts'));
+
+    it('Make sure that we get Timeline data', async () => {
+      await retry.try(async () => {
+        const resp = await supertest
+          .post('/internal/search/securitySolutionTimelineSearchStrategy/')
+          .set('kbn-xsrf', 'true')
+          .set('Content-Type', 'application/json')
+          .send({
+            defaultIndex: ['auditbeat-*'],
+            docValueFields: [
+              {
+                field: '@timestamp',
```

Comment:
The `docValueFields` definition is fairly long. Also, it's unclear if there is any special meaning to the data. The data is duplicated in the second test. I think this hinders readability. Perhaps you could define the data in one place and add a comment explaining the meaning. For example you could write `// a valid value for docValueFields must be passed.`

Reply:
Good point. I've added a constant for docValueFields and link it to the document.